test/utils/skipLogic.js

```diff
@@ -320,30 +326,23 @@ tap.test('.skipLogic()', { autoend: true }, (t) => {
       }]
     }
 
-    skipLogicGroupCheck(scope, 'yes', check)
+    skipLogicGroupCheck(scope, check)
     t.notOk(scope.field.show)
 
     check.logicGate = 'or'
-    skipLogicGroupCheck(scope, 'yes', check)
-    t.ok(scope.field.show)
-    skipLogicGroupCheck(scope, 'no', check)
+    skipLogicGroupCheck(scope, check)
     t.ok(scope.field.show)
 
+    scope.form.var2.$modelValue = 'yes'
     check.logicGate = 'and'
-    check.group[1].value = 'yes'
-    skipLogicGroupCheck(scope, 'yes', check)
+    skipLogicGroupCheck(scope, check)
```

Comment:
i am not sure why you are deciding to change these testcases!!! they would still pass if you resolve my above reviews @tmvumbi2